### ISSUE TITLE
[Toolbar]: Title nested in Toolbar is not getting focussed using Screen Reader.

### ISSUE THREAD

**Issue by NandithaRamesh** [SAP/ui5-webcomponents-react#4418]
### Describe the bug

Title nested in Toolbar is not getting focussed using Screen Reader.

### Reproduction steps

1.Nest a ui5-webcomponents Title component within a Toolbar component
2.On using Screen Reader, the user is unable to focus on and read the Title.
3.Screen Reader used: JAWS Version 2023.2302.15
...

### Expected Behaviour

Title must be focussed so that Screen Reader can read the Title.

### Screenshots or Videos

![recording_AdobeExpress](https://user-images.githubusercontent.com/129138100/228123171-b10d43f8-8e9f-4ea7-b2a4-a8ba10edac1e.gif)

<img width="584" alt="Screenshot 2023-03-28 092004" src="https://user-images.githubusercontent.com/129138100/228123539-c278d615-fa58-48c5-985b-23b2dadb1c6a.png">

### UI5 Web Components for React Version

1.10.3

### UI5 Web Components Version

1.10.3

### Browser

Chrome

### Operating System

Windows

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @NandithaRamesh 

In general, only interactive elements should be focusable. You can find out more about this [here](https://webaim.org/techniques/keyboard/) for example. 
Most screen readers have different reading modes for handling different kind of applications or sections and if you want to control what is announced to the user, then you can also set `aria-` attributes yourself. (The same goes for the `tab-index` but I wouldn't recommend adding it as explained above)

**Comment by NandithaRamesh** [comment]
Hi @Lukas742 

Thank you for your help and your suggestions
